src/pages/CreateUserPage.jsx

```diff
@@ -575,51 +577,51 @@ function CreateUserPage(props) {
-                            <div className="row justify-content-center justify-content-lg-start gx-2 mt-0">
-                                <div className="col-3 col-md-2">
-                                    <TextButton
-                                        text={isEditing ? 'Concluir' : 'Criar'}
-                                        hsl={[97, 43, 70]}
-                                        onClick={() => {
-                                            showAlert({
-                                                title: `Tem certeza que deseja ${isEditing ? 'editar' : 'criar'} o usuário?`,
-                                                dismissHsl: [355, 78, 66],
-                                                dismissText: 'Não',
-                                                actionHsl: [97, 43, 70],
-                                                actionText: 'Sim',
-                                                dismissible: true,
-                                                actionOnClick: () => {
-                                                    formRef.current.requestSubmit();
-                                                },
-                                            });
-                                        }}
-                                    />
-                                </div>
-                                {isEditing && (
-                                    <div className="col-3 col-md-2">
-                                        <TextButton
-                                            text={'Excluir'}
-                                            hsl={[355, 78, 66]}
-                                            onClick={() => {
-                                                showAlert({
-                                                    title: `Tem certeza que deseja excluir o usuário?`,
-                                                    dismissHsl: [97, 43, 70],
-                                                    dismissText: 'Não',
-                                                    actionHsl: [355, 78, 66],
-                                                    actionText: 'Sim',
-                                                    dismissible: true,
-                                                    actionOnClick: () => {
-                                                        deleteUser();
-                                                    },
-                                                });
-                                            }}
-                                        />
+                            <div className="row justify-content-center font-barlow gx-0">
+                                <div className="col col-md-10 d-flex flex-column h-100 p-4">
+                                    <div className="row justify-content-center justify-content-lg-start gx-2 mt-0">
+                                        <div className="col-3 col-md-2">
```

Comment:
```suggestion
                                        <div className="col-4 col-xl-3">
```
![image](https://github.com/user-attachments/assets/3fd8f5d8-6734-4c25-97d3-b70e93a89456)

Reply:
- Corrigido para `col-5 col-sm-3 col-xl-2`;